Imports are messed up

Import in source code and unit tests should be cleaned up with eclipse.

I think resolving this would be enough

```
[INFO] Compiling 1 source files to /Users/marco/.TheSparkBox/data/cloud-insight/cinsight/target/test-classes at 1493908028566
[WARNING] /Users/marco/.TheSparkBox/data/cloud-insight/cinsight/src/test/scala/se/uu/it/cinsight/CloudInsightTest.scala:5: warning: imported `CloudInsight' is permanently hidden by definition of class CloudInsight in package cinsight
[WARNING] import se.uu.it.cinsight.CloudInsight;
[WARNING]
```

Removing the import should be enough

--- a/cinsight/src/test/scala/se/uu/it/cinsight/CloudInsightTest.scala
+++ b/cinsight/src/test/scala/se/uu/it/cinsight/CloudInsightTest.scala
@@ -2,7 +2,6 @@ package se.uu.it.cinsight
 
 import scala.collection.immutable.List
 import scala.math.sqrt
-import se.uu.it.cinsight.CloudInsight;
 
 import org.apache.spark.SharedSparkContext
 import org.apache.spark.mllib.linalg.Vectors

